src/index.spec.js

```diff
@@ -332,44 +336,68 @@ describe('Dropzone', () => {
 
     it('should set proper dragActive state on dragEnter', () => {
       const dropzone = mount(
-        <Dropzone />
+        <Dropzone {...classConfig}>
+          {({ isDragActive, isDragReject }) => {
+            if (isDragReject) {
+              return 'Rejected';
+            }
+            if (isDragActive) {
+              return 'Active';
+            }
+            return 'Empty';
+          }}
+        </Dropzone>
```

Comment:
I'm wondering if we could just render some simple component with those props and then in the test check if it received the proper props. I think this is what we should test not the classNames — they are just implementation detail.